resources/acsl/digitalelectronics.md

```diff
@@ -9,12 +9,321 @@
 
 # Introduction
 
-Digital electronics is a way to visualize boolean algebra using drawn-out circuits.
+Digital electronics is a way to visualize boolean algebra using drawn-out circuits. The same laws of boolean algebra apply and truth tables are still very useful when analyzin them.
 
 <br>
 
 # Definitions
 
+<table class="wikitable" style="text-align: left">
+
+<tr>
+<th><b>NAME</b>
+</th>
+<th><b>GRAPHICAL SYMBOL</b>
+</th>
+<th><b>ALGEBRAIC EXPRESSION</b>
+</th>
+<th><b>TRUTH TABLE</b>
+</th></tr>
+<tr>
+<th><b>BUFFER</b>
+</th>
+<td> <a href="https://www.categories.acsl.org/wiki/images/a/a6/Buffer-gate-en.svg" class="image"><img alt="Buffer-gate-en.svg" src="https://www.categories.acsl.org/wiki/images/a/a6/Buffer-gate-en.svg" width="128" height="46" srcset="www.categories.acsl.org/wiki/images/thumb/a/a6/Buffer-gate-en.svg/192px-Buffer-gate-en.svg.png 1.5x, www.categories.acsl.org/wiki/images/thumb/a/a6/Buffer-gate-en.svg/256px-Buffer-gate-en.svg.png 2x" /></a>
```

Comment:
don't use ACSL images; download the images and throw them into the `res` folder under the proper folder.

I'll show you how to do this later. Just send me a message